src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/monitoring/IncompleteEnvelopesTask.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.tasks.monitoring;
+
+import net.javacrumbs.shedlock.core.SchedulerLock;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+@Component
+@ConditionalOnProperty(prefix = "monitoring.incomplete-envelopes", name = "enabled")
+public class IncompleteEnvelopesTask {
+
+    private static final Logger log = LoggerFactory.getLogger(IncompleteEnvelopesTask.class);
+
+    @SuppressWarnings("all") // tmp until implemented
+    public IncompleteEnvelopesTask(
+        // autowire repository
+    ) {
+        // empty constructor
+    }
+
+    @Scheduled(cron = "${monitoring.incomplete-envelopes.cron}")
+    @SchedulerLock(name = "incomplete-envelopes-monitoring")
+    public void run() {
+        log.info("Checking for incomplete envelopes");
+
+        // get data and log here
+
+        log.debug("Finished checking for incomplete envelopes");
```

Comment:
is this ever logged?

Reply:
at the moment this method stands for wrapper what's coming. naturally we should have debugs but apparently it's not in team behaviour as inspected over the time. this log is not essential and just marks the fact job has finished. would argue with first one too, but, like i said, team is in favour of infos